gapis/api/vulkan/find_issues.go

```diff
@@ -286,18 +284,19 @@ func (t *findIssues) Flush(ctx context.Context, out transform.Writer) error {
 			var issue replay.Issue
 			msg := eMsg.GetMsg()
 			label := eMsg.GetLabel()
-			if int(label) < t.numInitialCmds {
+			issue.Command = api.CmdID(label)
+			issue.Severity = service.Severity(uint32(eMsg.GetSeverity()))
+
+			if label == 0 {
 				// The debug report is issued for state rebuilding command
 				// TODO: Fix all the errors reported for initial commands.
-				issue.Command = api.CmdID(0)
-				issue.Error = fmt.Errorf("[State rebuilding command, linearized ID: %d]: %s", label, msg)
+				issue.Error = fmt.Errorf("[State rebuilding command : %s", msg)
```

Comment:
We don't have the id/index of the state rebuilding command that triggers an error. Would there be a way to keep this information?
ALso, missing closing `]`

Reply:
I will add the `]`.

We don't have because it actually does not mean anything. They are the commands produced by AGI to rebuild state rather than the real commands. Currently, they have a command id 0. Please refer @pmuetschard for further information.